docs/developer-docs.md

```diff
@@ -0,0 +1,96 @@
+EasyEngine Developer Docs
+=========================
+
+## Requirements
+
+* Git
+* PHP
+* Composer
+
+Before starting refer the [core repository structure](core-repo-structure.md) and the [structure for multiple repositories](structure-for-multiple-repos.md).
+
+## Steps for working on core repository
+
+1. Fork the EasyEngine core repository, clone it locally and checkout the development branch.
+```bash
+git clone git@github.com:your-username/easyengine.git && git checkout develop-v4 
+```
+2. Run `composer install` in the core repository after this.  
+
+3. Make required changes and check them by running it locally using the following from the easyengine repository root:
+
+```bash
+$ ./bin/ee command
+```
+
+## Steps for working on existing commands
+
+1. Clone the EasyEngine core repository locally and checkout the development branch.
+```bash
+git clone git@github.com:EasyEngine/easyengine.git && git checkout develop-v4 
+```
+
+2. Fork the command you want to work on.
```

Comment:
Give site-command with the url - https://github.com/EasyEngine/site-command/ as example throughout the docs